src/Angular.js

```diff
@@ -1457,7 +1510,9 @@ function bindJQuery() {
   }
 
   // bind to jQuery if present;
-  jQuery = window.jQuery;
+  var jqName = jq();
+  jQuery = jqName != null ? window[jqName] : window.jQuery;
```

Comment:
I've missed it before: an empty `ng-jq` attribute should mean "use jqLite" and not "use the native detection algorithm" as it's done now. Otherwise it'll be impossible for people to opt in to jqLite if jQuery is included on the site which is one of the major points of this functionality. :)

Refs the [above comment](https://github.com/angular/angular.js/pull/10761/files#r23366361) by @gkalpak.